spec/models/vote_spec.rb

```diff
@@ -100,7 +86,7 @@
     end
 
     context "when the user has not voted before" do
-      before(:each) { allow(Vote).to receive(:voted?).and_return (false) }
+      before(:each) { allow_any_instance_of(subject_class).to receive(:vote_by).and_return (nil) }
```

Comment:
Just wondering. Why do we need to stub the vote_by method?. `allow_any_instance_of(:Class)` should be used with caution.